Makefile

```diff
@@ -142,51 +145,26 @@ protos := \
 	proto/private/server/journal/journal.proto \
 	proto/spire/common/common.proto \
 
-serviceprotos := \
-	proto/private/test/catalogtest/test.proto \
+api-protos := \
 	proto/spire/api/registration/registration.proto \
+
+plugin-protos := \
+	proto/private/test/legacyplugin/someplugin.proto \
 	proto/spire/common/plugin/plugin.proto \
-	proto/spire/hostservice/server/agentstore/v0/agentstore.proto \
-	proto/spire/hostservice/server/identityprovider/v0/identityprovider.proto \
-	proto/spire/hostservice/common/metrics/v0/metrics.proto \
 	proto/spire/plugin/agent/keymanager/v0/keymanager.proto \
 	proto/spire/plugin/agent/nodeattestor/v0/nodeattestor.proto \
-	proto/spire/plugin/agent/svidstore/v0/svidstore.proto \
+	proto/spire/plugin/agent/svidstore/v1/svidstore.proto \
```

Comment:
Naming it `v0` in the first place was probably not the right thing. `v0`'s, in my mind, are reserved for interfaces published pre-sdk. Considering there are no consumers, I thought it safe to just correct this now. It could have been done later, to your point.